### PR TITLE
M-net: add mined subdomains

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

File: src/chrome/content/rules/M-net.de.xml [hunk 1]
```diff
@@ -1,9 +1,82 @@
 <ruleset name="M-net Telekommunikations GmbH" default_off="broken">
```
Comment (J0WI) [inline src/chrome/content/rules/M-net.de.xml]: Note that this is still `default_off`

Comment (numismatika) [inline src/chrome/content/rules/M-net.de.xml]: At the tine the request was made the mixed content broke the page and a few places had redirects to plain. I will check it again.

Comment (numismatika) [inline src/chrome/content/rules/M-net.de.xml]: So re-enabled.
I only could find a small mixed-content issue with a tracking element on the board. Shouldn't be an issue.

**Comment by J0WI** [comment]
Travis reports now a bunch of certificate issues. 😿

**Comment by Folant** [comment]
@J0WI https://github.com/EFForg/https-everywhere/pull/7492#issuecomment-281836532

**Comment by numismatika** [comment]
the ones with the broken server hello are probably explained with the linked comment,bu t there are also a few other errors for domains that are broken or seemingly internal.

**Comment by numismatika** [comment]
@J0WI could you fix the label now that checks are green?

**Pull request merged**